src/aap_eda/core/models/activation.py

```diff
@@ -113,6 +117,12 @@ class Meta:
         related_name="+",
     )
 
+    def save(self, *args, **kwargs):
+        super().save(*args, **kwargs)
+        if not self.organization:
+            self.organization = Organization.objects.get_default()
+            super().save(update_fields=["organization"])
```

Comment:
Interesting approach. It should unblock some other work before serializer changes happen.